### ISSUE TITLE
Support SLA options in execute API

### ISSUE THREAD

**Issue by juhoautio** [azkaban/azkaban#2509]
At the moment SLA options can be only set for schedules, because in the API it's only implemented in `ajaxSetSla` of `ScheduleServlet`.

However, SLA options are stored in `ExecutionOptions` of `ExecutableFlow`, so internally Azkaban execution model would naturally support SLA also for non-scheduled executions.

Would you be willing to review a contribution (pull request) where `ajaxExecuteFlow` of `ExecutorServlet` is extended to also allow passing SLA options as request params?

**Comment by djaiswal83** [comment]
This definitely makes sense. Will run this by the team and update here.

**Comment by juhoautio** [comment]
@djaiswal83 thanks, any update?

**Comment by djaiswal83** [comment]
@juhoautio Due to priorities I am not sure if we can pick this up anytime soon.

**Comment by juhoautio** [comment]
@djaiswal83 just to confirm, do you mean that you don't even have time to review the PR at this point if I make one?

( + maybe some QA work that is not visible to the public )

**Comment by djaiswal83** [comment]
No, I meant that it is low in our priorities to pick this up. If there is a PR, we are happy to look into.

**Comment by juhoautio** [comment]
Thanks, nice! That was my question actually:

> Would you be willing to review a contribution (pull request)

**Comment by djaiswal83** [comment]
Oh yes!
